common/containers/Tabs/SendTransaction/index.tsx

```diff
@@ -415,15 +415,15 @@ export class SendTransaction extends React.Component<Props, State> {
     const query = queryString.parse(searchStr);
     const to = getParam(query, 'to');
     const data = getParam(query, 'data');
-    // FIXME validate token against presets
     const unit = getParam(query, 'tokenSymbol');
+    const token = this.props.tokens.find(x => x.symbol === unit);
     const value = getParam(query, 'value');
-    let gasLimit = getParam(query, 'gas');
+    let gasLimit = getParam(query, 'gaslimit');
```

Comment:
Note to self: need to confirm if the legacy query param spec was indeed `gasLimit` instead of `gas`.

Reply:
confirmed

Reply:
both are supported https://github.com/kvhnuke/etherwallet/blob/mercury/app/scripts/controllers/sendTxCtrl.js#L28